src/lib/components/schedule/schedule-form-view.svelte

```diff
@@ -31,13 +33,18 @@
   let namespace = $page.params.namespace;
   let scheduleId = $page.params.schedule;
 
-  let title = schedule ? 'Edit Schedule' : 'Create Schedule';
-  let loadingText = schedule ? 'Editing Schedule...' : 'Creating Schedule...';
-  let backTitle = schedule ? 'Back to Schedule' : 'Back to Schedules';
+  let title = translate('schedules', schedule ? 'edit' : 'create');
+  let loadingText = translate('schedules', schedule ? 'editing' : 'creating');
```

Comment:
'edit' and 'create' seem like a common.json term, not specific to schedules

Reply:
nvm it's Creating Schedules...

Reply:
The copy here is "Edit Schedule", and "Create Schedule", so we can make them common if we remove "Schedule", just didn't want to change existing copy.

Reply:
Yup realized that looking at the json, it's totally fine. We can change next time there is a 'Create text'.